.pipelines/templates/PackageBuild.yml

```diff
@@ -0,0 +1,13 @@
+parameters:
+  - name: isCheckBuild
+    type: string
```

Comment:
Unfortunately I couldn't use the `boolean` type here, if I wanted to use the matrix strategy for running this template in the `PackageBuildPRCheck.yml` pipeline. The issue is that templates are "inlined" during pipeline startup, when the value of `$(runCheck)` is not known, yet, and the pipeline complains that `$(runCheck)` is neither `true` nor `false`.